raven-log4j2/src/main/java/com/getsentry/raven/log4j2/SentryAppender.java

```diff
@@ -234,6 +245,10 @@ protected Event buildEvent(LogEvent event) {
                 .withLevel(formatLevel(event.getLevel()))
                 .withExtra(THREAD_NAME, event.getThreadName());
 
+        if (!Strings.isNullOrEmpty(serverName)) {
+            eventBuilder.withServerName(serverName.trim());
```

Comment:
This is consistent with the other implementations, but seems like it really should be implemented in the mutator method and not here.